docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,34 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+Return all tables form the database. It shows databse name and whether a table is temporary.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} dbname] [LIKE 'pattern']
```

Comment:
refer [tables.scala](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala#L743) for syntax